src/random_lcg.F90

```diff
@@ -53,6 +54,21 @@ function prn() result(pseudo_rn)
   end function prn
 
 !===============================================================================
+! GET_PRN_AHEAD generates a pseudo-random number which is 'n' times ahead from
+! current seed.
+!===============================================================================
+
+  function get_prn_ahead(n) result(pseudo_rn)
```

Comment:
And there is also a `prn_skip`. These three, `prn_ahead`, `prn_skip_ahead ` and `prn_skip` seem a little confusing. Do you have more suggestions? @paulromano